db/db.test.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+describe('DB Test', () => {
+  beforeAll((done) => {
+    mongoose.connect('mongodb://localhost/data');
```

Comment:
You may want to consider using the "dotenv" npm package and a .env file to store your MongoDB server address.  That way it isn't hardcoded as a string in your test & in your mongodb.js file (what if you update one but forget to update the other?)

Reply:
thanks Mark, I will definitely look into this!